AENS.md

```diff
@@ -3,7 +3,7 @@
 
 ## Abstract
 
-This document describes the first draft of the Aeternity Naming System,
+This document describes the first draft of the AEternity Naming System,
```

Comment:
wut ?

Reply:
I don't get it ? You want to emphasise on abbreviation? Even that does not makes much sense to me :)

Reply:
ahh I think this was updated. should be fixed already. but I will double check.